scripts/core/exp-cooperative-kernels.yml

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Unified Runtime Experimental APIs for Cooperative Kernels"
+ordinal: "99"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: |
+      The extension string which defines support for cooperative-kernels
+      which is returned when querying device extensions.
+name: $X_COOPERATIVE_KERNELS_EXTENSION_STRING_EXP
+value: "\"$x_exp_cooperative_kernels\""
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Enqueue a command to execute a cooperative kernel"
+class: $xEnqueue
+name: CooperativeKernelLaunchExp
```

Comment:
This function is identical to `urKernelLaunch`.  The actual difference is in the kernel itself?  And given the `urSuggestMaxCooperativeGroupCountExp` entry point we can presumably extract that info from the kernel.

Why not have the use of co-operative groups be a property of the kernel and just reuse `urKernelLaunch`?

Reply:
I see, I missed the distinction that device code may select a different option based on how it was launched.  I'll resolve this convo.